versioning_function.sql

```diff
@@ -117,9 +118,9 @@ BEGIN
     END IF;
 
     -- If we are not including the current version in the history, we need to check if the current version is valid
-    IF include_current_version_in_history <> 'true' THEN
```

Comment:
is this condition supposed to be gone altogether?

Reply:
Yes, I think that was an incorrect condition. We still want to perform the checks and update conflict mitigation in the case we we are performing an update or delete, regardless of if we are including current version in the history.

Reply:
The original PR for the current version in history feature was setting it like this but it won't cater for updates when including current version in history. These are still a valid type of update that we want to perform checks and resolve update conflicts with.

Reply:
Let me update the comment as such above the IF